data/en/get_started.json

```diff
@@ -48,6 +48,13 @@
           "title": "Install CDAP in Mission-Critical Environments",
           "description": "If you are ready to install CDAP on your Hadoop cluster, in your Development, Testing, or Production environments, the following options are available to you: To add CDAP to your Cloudera Manager installations, using the CDAP Custom Service Descriptor (CSD) is the way to go. To use CDAP within your Hortonworks Data Platform (HDP) environment, use Apache Ambari (note that Apache Ambari can only be used to add CDAP to an existing Hadoop cluster). If you are using MapR, BigTop or the Apache distribution of Hadoop, we provide RPM and Debian distributions of CDAP to support your varied installation needs.",
           "items": [
+            {
+              "title": "Cloud Data Fusion on GCP",
+              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin",
```

Comment:
Cloud Data Fusion (powered by CDAP) is a fully-managed, cloud native, enterprise data integration service on GCP for quickly building and managing data pipelines.

Reply:
Done. We make this sentence a little bit shorter because it breaks layout.

```
Cloud native, enterprise data integration service on GCP for quickly building and managing data pipelines.
```

Is it ok?

Reply:
Actually, powered by CDAP is kind of important.

How about 
```
Fully-managed, cloud-native, enterprise data integration service powered by CDAP for quickly building and managing data pipelines
```

Reply:
If its not possible, can we fit "powered by CDAP" in the title?

Title: Google Cloud Data Fusion (powered by CDAP)
Description: Fully-managed, cloud-native, enterprise data integration service for quickly building and managing data pipelines

?

Reply:
Done. Please check